### ISSUE TITLE
Unable to start Web server

### ISSUE THREAD

**Issue by nitheshreddy** [kzahel/web-server-chrome#56]

**Comment by nitheshreddy** [comment]
I just installed the app and selected "Also on internet" option. That is when i tried turning off and turning it back on. I was waiting for a while but it didn't start.

**Comment by nitheshreddy** [comment]
![image](https://cloud.githubusercontent.com/assets/8356260/15412520/0f347bae-1ddd-11e6-8bc1-e26c69f35f6f.png)

**Comment by JoseRenan** [comment]
I had the same problem :/

**Comment by kzahel** [comment]
Hmm something is going wrong talking to your router most likely. If you can go to chrome://inspect and click on the background page for web server chrome and paste the console log window that would help

**Comment by nitheshreddy** [comment]
![image](https://cloud.githubusercontent.com/assets/8356260/15436811/80e978c6-1e78-11e6-9c02-cd9e3accc826.png)

**Comment by nitheshreddy** [comment]
It just stays there and would not start :/

**Comment by kzahel** [comment]
I think I see the issue.
To clarify though, it will start if you turn off "Also on internet" and completely restart the app? (e.g. by turning off background mode and closing the window, or rebooting)

**Comment by nitheshreddy** [comment]
Yes, it is working when i turn off "Also on Internet" option. But how to make it work with selecting this option.

**Comment by kzahel** [comment]
You might want to check your router settings (usually by going to http://192.168.1.1) and making sure upnp is turned on. You could also manually forward a port to your computer's ip. But it's possible there's some other bug happening that I could fix. I can add more logging to determine what is going wrong in the next update.

**Comment by marcacohen** [comment]
I'm seeing the same problem, reproducibly.

**Comment by ganggreen777** [comment]
Same problem here,and posted this very problem, w/o a response back

**Comment by kzahel** [comment]
I'll get to fixing this on Monday

**Comment by kzahel** [comment]
I put out an update to the web store 0.4.1 that you should be able to try out in an hour or two when it auto-updates. I added an option in the advanced settings to do "Verbose logging" which you can view at chrome://inspect under Extensions

**Comment by kzahel** [comment]
Please try to disable the "available on internet" if you encounter this problem.

**Comment by nitheshcse** [comment]
So, Is it not possible to access these URLs from outside local network?

**Comment by kzahel** [comment]
It is, but only if your router supports upnp port mapping. Also I only tested on a few routers so I can't guarantee it works everywhere. Also if there's more than one computer on the same network trying to do it, there can be a conflict and it will only work for one computer...
You can look at the log file to see the status by opening a new tab and going to the url "chrome://inspect" and then look for the background page for web server chrome, look for things related to UPNP...

**Comment by bj32161** [comment]
Hi,
I also have problem. My case I setup Kiosk mode to run my own App(Appspace) and also install your Web Server Extension as well. I did setup run on background and access on local network. When I reboot my app show up first as it is Kiosk App. I tried to access this chrome-box http://x.x.x.x:8887/index.html from my notebook. It now show anything. 
Please help. Web server for chrome is very good by the way. Hope this issue will fix.
Thank you,
Panya

**Comment by kzahel** [comment]
@bj32161 I think I saw your message on the app support page. I think when you run a Kiosk app, it only runs that app, and ignores all other apps, even if they're set to run on startup. That means you'll need to combine the apps into a single app. You might want to look at https://github.com/matt-cook/kiosk which is able to run as kiosk and use web server chrome also.

**Comment by bj32161** [comment]
@kzahel Thank you for your advise. Unfortunately, the kiosk app developed by another company which I can't bundle this this App. I just looking work around.

**Comment by kzahel** [comment]
Does the android appspace support a similar feature (hosting local content?)
It might be possible for them to bundle in WSC. Maybe they have some kind of product feature suggestion page that you could post to.

**Comment by ajilpappachan** [comment]
![image](https://user-images.githubusercontent.com/29731499/130673140-ee41268c-807a-438a-9148-57d98689ec35.png)

I am getting this error. I tried turning off also available on internet, available on local network, use https, and pretty much every setting I turned on. But still stuck.

**Comment by kzahel** [comment]
The original Chrome App is no longer supported. The app has been rebuilt from scratch — see https://ok200.app for the new versions and to sign up for updates.
